lib/Themes/OverrideDefaultTheme.php

```diff
@@ -99,12 +100,14 @@ public function getCSSVariables(): array {
 		$ionosVariables = [
 			'--ion-color-main-background' => $ionColorMainBackground,
 			'--ion-color-primary' => $ionColorPrimary,
-			'--ion-color-secondary' => $ionColorSecondary,
+			'--ion-color-secondary' => $ionColorBlueB8,
 			'--ion-color-blue-b1' => $ionColorBlueB1,
 			'--ion-color-blue-b2' => $ionColorBlueB2,
 			'--ion-color-blue-b4' => $ionColorBlueB4,
 			'--ion-color-blue-b5' => $ionColorBlueB5,
+			'--ion-color-blue-b6' => $ionColorBlueB6,
 			'--ion-color-blue-b7' => $ionColorBlueB7,
+			'--ion-color-blue-b8' => $ionColorBlueB8,
```

Comment:
why do we need this variable, if it's the same as `--ion-color-secondary`? Can't we just use that instead?

Reply:
Because `--ion-color-secondary` could change to be `#ff8800` at some point
But `--ion-color-blue-b8` should never be something else than blue.

Reply:
Ah... of course. Makes a lot of sense now. :facepalm: